fail to install faiss for arm64-windows due to "__lzcnt64"

E:\vcpkg>vcpkg install faiss
Computing installation plan...
The following packages will be built and installed:
    faiss:arm64-windows@1.7.4
Detecting compiler hash for triplet arm64-windows...
Compiler found: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.40.33807/bin/Hostarm64/arm64/cl.exe
Restored 0 package(s) from C:\Users\Administrator\AppData\Local\vcpkg\archives in 80.9 us. Use --debug to see more details.
Installing 1/1 faiss:arm64-windows@1.7.4...
Building faiss:arm64-windows@1.7.4...
-- Using cached facebookresearch-faiss-v1.7.4.tar.gz.
-- Cleaning sources at E:/vcpkg/buildtrees/faiss/src/v1.7.4-b01657e44a.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source E:/vcpkg/downloads/facebookresearch-faiss-v1.7.4.tar.gz
-- Applying patch fix-dependencies.patch
-- Using source at E:/vcpkg/buildtrees/faiss/src/v1.7.4-b01657e44a.clean
-- Found external ninja('1.11.0').
-- Configuring arm64-windows
-- Building arm64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: "C:/Program Files/CMake/bin/cmake.exe" --build . --config Debug --target install -- -v -j13
    Working Directory: E:/vcpkg/buildtrees/faiss/arm64-windows-dbg
    See logs for more information:
      E:\vcpkg\buildtrees\faiss\install-arm64-windows-dbg-out.log

Call Stack (most recent call first):
  installed/arm64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/arm64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/faiss/portfile.cmake:30 (vcpkg_cmake_install)
  scripts/ports.cmake:192 (include)

C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1440~1.338\bin\HOSTAR~1\arm64\cl.exe   /TP -DFAISS_MAIN_LIB -DFINTEGER=int -Dfaiss_EXPORTS -IE:\vcpkg\buildtrees\faiss\src\v1.7.4-b01657e44a.clean -external:IE:\vcpkg\installed\arm64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd -openmp /showIncludes /Fofaiss\CMakeFiles\faiss.dir\AutoTune.cpp.obj /Fdfaiss\CMakeFiles\faiss.dir\ /FS -c E:\vcpkg\buildtrees\faiss\src\v1.7.4-b01657e44a.clean\faiss\AutoTune.cpp
E:\vcpkg\buildtrees\faiss\src\v1.7.4-b01657e44a.clean\faiss/impl/platform_macros.h(59): error C3861: '__lzcnt64': identifier not found
ninja: build stopped: subcommand failed.
[fail_install_faiss.zip](https://github.com/user-attachments/files/16868983/fail_install_faiss.zip)

--- a/ports/faiss/vcpkg.json
+++ b/ports/faiss/vcpkg.json
@@ -5,7 +5,7 @@
   "description": "Faiss is a library for efficient similarity search and clustering of dense vectors.",
   "homepage": "https://github.com/facebookresearch/faiss",
   "license": "MIT",
-  "supports": "!uwp & !osx & !x86 & !(arm & windows)",
+  "supports": "!uwp & !osx & !x86 & !(arm64 & windows)",
   "dependencies": [
     "lapack",
     "openblas",

--- a/ports/faiss/vcpkg.json
+++ b/ports/faiss/vcpkg.json
@@ -1,10 +1,11 @@
 {
   "name": "faiss",
   "version": "1.7.4",
+  "port-version": 1,
   "description": "Faiss is a library for efficient similarity search and clustering of dense vectors.",
   "homepage": "https://github.com/facebookresearch/faiss",
   "license": "MIT",
-  "supports": "!uwp & !osx & !x86",
+  "supports": "!uwp & !osx & !x86 & !(arm & windows)",
   "dependencies": [
     "lapack",
     "openblas",

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -329,7 +329,6 @@ epsilon:x64-uwp=fail
 evpp:arm-neon-android=fail
 evpp:arm64-android=fail
 evpp:x64-android=fail
-faiss:arm64-windows=fail
 fastrtps:arm-neon-android=fail
 fastrtps:arm64-android=fail
 fastrtps:x64-android=fail

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2618,7 +2618,7 @@
     },
     "faiss": {
       "baseline": "1.7.4",
-      "port-version": 0
+      "port-version": 1
     },
     "fakeit": {
       "baseline": "2.4.0",

--- a/versions/f-/faiss.json
+++ b/versions/f-/faiss.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1f23586d2052bbd58cac5b0db51ffcf4239117e9",
+      "version": "1.7.4",
+      "port-version": 1
+    },
+    {
       "git-tree": "e2f2e6695629b11c85f79e4fc8b962134396a953",
       "version": "1.7.4",
       "port-version": 0

--- a/versions/f-/faiss.json
+++ b/versions/f-/faiss.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "1f23586d2052bbd58cac5b0db51ffcf4239117e9",
+      "git-tree": "bc836dd883cc277c5b11a98e8155f53503ef37f9",
       "version": "1.7.4",
       "port-version": 1
     },

